src/main/java/com/trello/BoardPage.java

```diff
@@ -0,0 +1,102 @@
+package com.trello;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class BoardPage {
+
+    @FindBy (xpath = ".//button[contains(@aria-label, 'Open Member Menu')]")
+    private WebElement openMemberMenuButton;
+
+    @FindBy (xpath = ".//button[contains(@aria-label, 'Open Boards Menu')]")
+    private WebElement openBoardsMenuButton;
+
+    @FindBy (xpath =  ".//button[@data-test-id = 'header-member-menu-logout']")
+    private WebElement menuLogOutLink;
+
+    @FindBy (xpath = ".//input[@name= 'search-boards']")
+    public WebElement searchField;
+
+    @FindBy (xpath = ".//button[@data-test-id= 'header-boards-menu-create-board']")
+    private WebElement newBoardLink;
+
+    @FindBy (xpath = ".//input[@placeholder= 'Add board title']")
+    private WebElement newBoardTitleField;
+
+    @FindBy (xpath = ".//button[@data-test-id='create-board-submit-button']")
+    private WebElement newBoardSubmit;
+
+    @FindBy (xpath = ".//input[@class= 'board-name-input js-board-name-input']")
+    public WebElement boardTitle;
```

Comment:
ALL WebElements in Page classes should be **private**

Reply:
ok